noxfile.py

```diff
@@ -0,0 +1,132 @@
+import tempfile
```

Comment:
Hvorfor trenger vi hele nox oppsettet hvis det allerede finnes noe som støttes av pipenv?
https://pipenv-fork.readthedocs.io/en/latest/advanced.html#custom-script-shortcuts

Reply:
vi bruker nox fordi det oppretter virituelle miljø alt kjøres i, ikke pga eventuell script-funksjonalitet

Reply:
nox og poetry er good guys. Det er andre ting som er bra også, som feks pipenv. Det er for så vidt opp til den enkelte utvikler å, sidan vi ikkje har standardisert på dette. 
nox poetry comboen er sterkt inspirert av denne blogg-serien:
https://cjolowicz.github.io/posts/hypermodern-python-01-setup/